src/main/java/com/dougnoel/sentinel/steps/ImageSteps.java

```diff
@@ -27,22 +27,23 @@ public static void before(Scenario scenario) {
 	/**
 	 * Takes a screenshot of the given element and stores it for later comparison
 	 * @param elementName String name of the element to screenshot
-	 * @throws IOException
+	 * @throws IOException If the expected image to be created fails to save
 	 */
-    @When("^I take a screenshot of (?:the|a) (page|.*)$")
+    @When("^I take a screenshot of (?:the|a) (.*?)$")
     public static void storeScreenshotOfElement(String elementName) throws IOException {
-    	String imageFileName = scenario.getName()+ "_" + elementName + "_" + PageManager.getPage().getName() + ".png";
+		String outputFolder = "imageComparison/" + scenario.getName();
+    	String imageFileName = PageManager.getPage().getName() + "_" + elementName + "_EXPECTED" + ".png";
```

Comment:
Should there be an element of randomness (timestamp) in the name, or is it smart enough to overwrite on a rerun? Do we want to keep the old screenshots for records' sake?

Reply:
For right now it's smart enough to perform overwrites. Unless I'm horribly mistaken.

Archiving for later is not currently implemented, but we can look into it later.